docs/platform/howto/saml/setup-saml-auth0.rst

```diff
@@ -42,15 +42,15 @@ Configure SAML on Auth0
 
 #. In the **Settings** section for the Application Callback URL, remove the existing configuration and add the following field mapping configuration:
 
-.. code-block:: shell
-
-   {
-     "email": "email",
-     "first_name": "first_name",
-     "identity": "email",
-     "last_name": "last_name",
-     "mapUnknownClaimsAsIs": true
-   }
+    .. code-block:: shell
```

Comment:
This code block start should align with the text to avoid the extra frame around here. 

![image](https://user-images.githubusercontent.com/108724024/231792680-6127bea6-d8fc-453b-986b-aa174a12ff81.png)